src/game/SpellEffects.cpp

```diff
@@ -1151,7 +1151,7 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
                 }
                 case 32027:                                 // Expedition Flare
                 {
-                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
+                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
                         m_caster->CastSpell(m_caster, (urand(0, 1) ? 32029 : 32030), true, m_CastItem);
```

Comment:
Actually I very much think that the spell will indeed work if cast alone; it just won't cast if the cast is conditioned on m_CastItem not being a nullptr, because it is in this case (according to the PR'er himself) - therefor we have to remove the check, and if the CastItem is anyways a nullptr, there is no reason to send it along.

CastItem works fine in most dummies because the item casting the spell usually doesn't get consumed on use? maybe?